src/layout.rs

```diff
@@ -246,23 +252,41 @@ pub struct Layout {
     segment_size: SegmentSize,
 }
 
-impl Default for Layout {
-    fn default() -> Layout {
-        Layout::new(Direction::Vertical, [])
-    }
-}
```

Comment:
Do we make sure with any tests that the default `Default` implementation and this implementation produce the same results?

Reply:
Yes now in 9c732ca9